package.json

```diff
@@ -83,7 +83,7 @@
     "fast-plist": "0.1.2",
     "glob": "^5.0.13",
     "gulp": "^4.0.0",
-    "gulp-atom-electron": "^1.21.1",
+    "gulp-atom-electron": "deepak1556/gulp-atom-electron#update_helper_rename",
```

Comment:
@deepak1556 isn't that released as `1.22.0`?

Reply:
yup should update it.